articles/azure-government/documentation-government-services-monitoringandmanagement.md

```diff
@@ -153,6 +153,18 @@ Add-AzureRmMetricAlertRule -Name vmcpu_gt_1 -Location "USGov Virginia" -Resource
 
 For more information on using PowerShell, see [public documentation](../azure-monitor/platform/powershell-quickstart-samples.md).
 
+## Application Insights
+The Azure Application Insights service uses a number of IP addresses. You might need to know these addresses if the app that you are monitoring is hosted behind a firewall.
+
+[!NOTE] Although these addresses are static, it's possible that we will need to change them from time to time. All Application Insights traffic represents outbound traffic with the exception of availability monitoring and webhooks which require inbound firewall rules.
```

Comment:
You need to change this line slightly. In order for the note to render properly it needs to be in this form with the right angle brackets:


```
> [!NOTE]
> Although these addresses are static, it's possible  that we will need to change them from time to time. All Application Insights traffic represents outbound traffic with the exception of availability monitoring and webhooks which require inbound firewall rules.
```

Reply:
Hi Michael, I adjusted the Note format. Can this PR be merged? Thanks

Reply:
Yep, thank you so much!

Reply:
Thank you!